build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -65,19 +65,19 @@ endfunction()
 # Parameters:
 #   TARGET_NAME: The target name to be created for this module.
 #   SOURCE: Source TF model direcotry
-#   ENTRY_FUNCTION: The entry function name for the input module.
+#   IMPORT_FLAGS: Flags to include in the import command.
```

Comment:
What's the reason we want to replace `ENTRY_FUNCTION` with `IMPORT_FLAGS` here?

`ENTRY_FUNCTION` is probably not a right design choice but it seems like in this change we still only specify `--tf-savedmodel-exported-names={ENTRY_FUNCTION} as import flags?`

Reply:
Using `--tf-savedmodel-exported-names={ENTRY_FUNCTION}` only works when the `SavedModel` has a `SignatureDef` and concrete function.

For example, MiniLM has the output below when querying the `SavedModel` with `saved_model_cli`. 

```
signature_def['serving_default']:
  The given SavedModel SignatureDef contains the following input(s):
    inputs['attention_mask'] tensor_info:
        dtype: DT_INT32
        shape: (1, 128)
        name: serving_default_attention_mask:0
    inputs['input_ids'] tensor_info:
        dtype: DT_INT32
        shape: (1, 128)
        name: serving_default_input_ids:0
    inputs['token_type_ids'] tensor_info:
        dtype: DT_INT32
        shape: (1, 128)
        name: serving_default_token_type_ids:0
  The given SavedModel SignatureDef contains the following output(s):
    outputs['last_hidden_state'] tensor_info:
        dtype: DT_FLOAT
        shape: (1, 128, 384)
        name: StatefulPartitionedCall:0
    outputs['pooler_output'] tensor_info:
        dtype: DT_FLOAT
        shape: (1, 384)
        name: StatefulPartitionedCall:1
  Method name is: tensorflow/serving/predict

Concrete Functions:
  Function Name: 'predict'
    Option #1
      Callable with:
        Argument #1
          input_ids: TensorSpec(shape=(1, 128), dtype=tf.int32, name='input_ids')
        Argument #2
          attention_mask: TensorSpec(shape=(1, 128), dtype=tf.int32, name='attention_mask')
        Argument #3
          token_type_ids: TensorSpec(shape=(1, 128), dtype=tf.int32, name='token_type_ids')
```

The `SavedModel` included in this PR doesn't have a concrete function, but still has a valid `SignatureDef`:

```
signature_def['serving_default']:
  The given SavedModel SignatureDef contains the following input(s):
    inputs['input_ids'] tensor_info:
        dtype: DT_INT32
        shape: (1, 384)
        name: input_ids:0
    inputs['input_mask'] tensor_info:
        dtype: DT_INT32
        shape: (1, 384)
        name: input_mask:0
    inputs['segment_ids'] tensor_info:
        dtype: DT_INT32
        shape: (1, 384)
        name: segment_ids:0
  The given SavedModel SignatureDef contains the following output(s):
    outputs['logits'] tensor_info:
        dtype: DT_FLOAT
        shape: (1, 384, 2)
        name: logits:0
  Method name is: tensorflow/serving/predict
```
In this case, `--tf-savedmodel-exported-names={ENTRY_FUNCTION}` won't work and we end up getting an empty MLIR file because that `ENTRY_FUNCTION` doesn't exist.

Reply:
I see, I missed that new added model.

There are some extra works to properly support import flags in the new benchmark suite (see the comment below). Feel free to open an issue and not support imported flags in the new suite. I can work on it later.

Reply:
I found that `--tf-savedmodel-exported-names=serving_default` works as intended as long as we correctly choose `--tf-import-type=savedmodel_v1`. Updated in #12170.